apps/user/models/team.go

```diff
@@ -0,0 +1,10 @@
+package models
+
+import "github.com/jinzhu/gorm"
+
+type Team struct {
+	gorm.Model
+	Name        string `gorm:"type:varchar(30);not null"`
+	Description string `gorm:"type:varchar"`
+	IsActive    bool   `gorm:"default:true; not null"`
```

Comment:
We is this named "IsActive" as compared to "Active" at other places ?

Reply:
changed to active